src/components/WeeklySummary/WeeklySummary.jsx

```diff
@@ -241,6 +256,21 @@ export class WeeklySummary extends Component {
     this.setState({ errors: errors || {} });
     if (errors) return;
 
+    let currentSubmittedCount = 0;
```

Comment:
Hey, @EvianTan I think you should add `currentSubmittedCount` as state and update it using `this.setState` and it will reflect the Updated submitted summaries instantly without reloading/ manual rerendering. Right now react is not re-rendering it because you are using a variable. use it as a state this will be a quick fix